code/game/machinery/doors/airlock.dm

```diff
@@ -664,6 +664,12 @@
 		else
 			to_chat(user, "It looks very robust.")
 
+	if(issilicon(user) && (!stat & BROKEN))
+		to_chat(user, "<span class='notice'>Shift-Click [src] to [ density ? "open" : "close"] it.</span>")
+		to_chat(user, "<span class='notice'>Ctrl-Click [src] to [ locked ? "raise" : "drop"] its bolts.</span>")
+		to_chat(user, "<span class='notice'>Alt-Click [src] to [ secondsElectrified ? "un-electrify" : "permanently electrify"] it.</span>")
+		to_chat(user, "<span class='notice'>Ctrl+Shift-Click [src] to [ emergency ? "disable" : "enable"] emergency access.</span>")
```

Comment:
Should be consistent about whether `+` or `-` is used to combine modifiers, ditto skittish

Reply:
I think I'll stick with `-`